docs/porting-to-ROS2.md

```diff
@@ -113,7 +115,12 @@ That's where differences start to show – I decided to make the `VehicleCmdGate
 
 
 ### Latched topics
-For each latched publisher, I just used `transient_local` QoS on the publisher, and we will need to do the same on all subscribers to that topic.
+For each latched publisher, you can use `transient_local` durability QoS on the publisher, e.g. when the history depth is 1:
+
+    rclcpp::QoS durable_qos{1};
```

Comment:
`transient_local()` returns the `rclcpp::QoS` object, so you can synthesize into a one-liner.

Reply:
True. Should I add it?

Reply:
I think it looks cleaner as a one-liner, but it's up to you.

Reply:
Done. If you want, feel free to merge.